packages/react-native-bridge/android/react-native-bridge/src/main/java/org/wordpress/mobile/WPAndroidGlue/GutenbergProps.kt

```diff
@@ -19,7 +19,8 @@ data class GutenbergProps @JvmOverloads constructor(
     val translations: Bundle,
     val isDarkMode: Boolean,
     val htmlModeEnabled: Boolean,
-    val enableEditorOnboarding: Boolean
+    val enableEditorOnboarding: Boolean,
+    val rawEditorSettings: String
```

Comment:
Since this comes as part of the same API as `editorTheme` I wonder if there is a way we can combine those.